Makefile

```diff
@@ -50,6 +50,10 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	sed '/^[dD]jango==/d' requirements/test.txt > requirements/test.tmp
 	mv requirements/test.tmp requirements/test.txt
 
+upgrade-package: ## update just one package to the latest usable release
+	@test -n "$(package)" || { echo "\nUsage: make upgrade_package package=...\n"; exit 1; }
+	$(MAKE) compile-requirements COMPILE_OPTS="--upgrade-package $(package)"
```

Comment:
I think you'll have to rework the make targets a bit to get this to work. In edx-platform, there's a `compile-requirements` target that does all the pip stuff, and `upgrade` and `upgrade-package` are wrappers that set additional arguments via `COMPILE_OPTS`. This isn't quite set up for that yet.

Reply:
Serves me right for copy-pasta.